### DISCUSSION TITLE
Welcome to pinia-colada Discussions!

### DISCUSSION THREAD

**Discussion by posva** [posva/pinia-colada#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drahkrub** [comment]
👋 Hi, I am a fullstack developer (spring/hibernate/vuejs) from Hannover, Germany.

**Comment by sabasayer** [comment]
Hello, I am a senior frontend developer with a deep passion for Vue and its amazing community. I’ve been using Vue since its early days and have built several large-scale projects with it.
I really love 💘  this package! I’ve created something similar in almost all of my projects, and I’d love the opportunity to contribute to it.
